### ISSUE TITLE
Keep track record of 'pending status' for data approval

### ISSUE THREAD

**Issue by janagombitova** [akvo/akvo-flow#1778]
**DRAFT TEXT** 
- the pending works better now than before - so we have an improvement already, but we will still lose the track record of someone submitting a pending issues once the step is approved. See below:

![pending gif1](https://cloud.githubusercontent.com/assets/12456965/19074203/6d62eeee-8a3f-11e6-993f-cab5f3ba6044.gif)
- I do not think this is an issue and my comment is only based on my hypothesis that the manager will want to have a full track record of everything
- I would suggest to leave the implementation as it is now, create a separate issue for this and see if in the 1st round of user testing SNV will give us such feedback or not (validate the hypothesis with real users)
